README.md

```diff
@@ -14,7 +14,7 @@
 </p>
 <p align="center">
     <a href="https://github.com/grafana/k6/releases">Download</a> ·
-    <a href="https://k6.io/docs">Documentation</a> ·
+    <a href="https://grafana.com/docs/k6/latest/">Documentation</a> ·
     <a href="https://community.k6.io/">Community Forum</a>
```

Comment:
Maybe change this also as part of this PR? :thinking:

Reply:
Yeah -good catch